dev-aws/kafka-shared-msk/energy-billing/modules.tf

```diff
@@ -2,7 +2,7 @@ module "billing_adapter" {
   source           = "../../../modules/tls-app"
   cert_common_name = "energy-billing/billing-adapter"
   produce_topics   = [kafka_topic.internal_energy_billing_billing_adapter_retry_1.name]
-  consume_topics   = [] # This would be the gentrack-adapter topic that supplies the billing JSON blobs
+  consume_topics   = ["energy-platform.gentrack.billing.events"]
```

Comment:
This fails linting due to this rule:

https://github.com/utilitywarehouse/tflint-ruleset-kafka-config/blob/main/rules/msk_app_topics.md.

The solution, as explained in the rule, is to have this exact consumer in the energy platform module for that topic consumption:

```
module "billing_adapter" {
  source           = "../../../modules/tls-app"
  cert_common_name = "energy-billing/billing-adapter"
  consume_topics   = ["energy-platform.gentrack.billing.events"]
}
```